repository/audit.go

```diff
@@ -49,8 +58,29 @@ type AuditLogDetail struct {
 }
 
 func NewAuditRepository(db *gorm.DB) *AuditRepository {
-	db.AutoMigrate(&AuditLog{}, &AuditLogDetail{})
-	return &AuditRepository{DB: db}
+	var duration time.Duration = 5 * time.Second
```

Comment:
Move this hardcoded `5 * time.Second`  into a global constant to be easier to change.